src/Storages/System/StorageSystemDDLWorkerQueue.cpp

```diff
@@ -0,0 +1,235 @@
+#include <algorithm>
+#include <filesystem>
+#include <string>
+
+#include "StorageSystemDDLWorkerQueue.h"
+
+#include <Columns/ColumnArray.h>
+#include <Interpreters/DDLWorker.h>
+
+#include <DataTypes/DataTypeArray.h>
+#include <DataTypes/DataTypeDateTime.h>
+#include <DataTypes/DataTypeEnum.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <Interpreters/Context.h>
+#include <Interpreters/evaluateConstantExpression.h>
+#include <Parsers/ASTExpressionList.h>
+#include <Parsers/ASTIdentifier.h>
+#include <Storages/SelectQueryInfo.h>
+#include <Common/ZooKeeper/ZooKeeper.h>
+
+#include <Parsers/ASTQueryWithOnCluster.h>
+#include <Parsers/ParserQuery.h>
+#include <Parsers/parseQuery.h>
+
+
+namespace fs = std::filesystem;
+
+enum Status
+{
+    active,
+    finished,
+    unknown,
+    errored
+};
+
+namespace DB
+{
+std::vector<std::pair<String, Int8>> getStatusEnumsAndValues()
+{
+    return std::vector<std::pair<String, Int8>>{
+        {"active", static_cast<Int8>(Status::active)},
+        {"finished", static_cast<Int8>(Status::finished)},
+        {"unknown", static_cast<Int8>(Status::unknown)},
+        {"errored", static_cast<Int8>(Status::errored)},
```

Comment:
Looks like enums in system tables uses CamelCase syntax (query_log/text_log)

Reply:
sure, changed to CamelCase.